core/src/main/java/org/mule/runtime/core/internal/context/DefaultMuleContext.java

```diff
@@ -357,6 +358,9 @@ public void start() throws MuleException {
       fireNotification(new MuleContextNotification(this, CONTEXT_STARTED));
       final org.mule.runtime.api.artifact.Registry apiRegistry = getApiRegistry();
       listeners.forEach(l -> l.onStart(this, apiRegistry));
+      Optional<ArtifactStoppedListener> optionalArtifactStoppedListener =
+          Optional.ofNullable(getRegistry().lookupObject(ARTIFACT_STOPPED_LISTENER));
+      optionalArtifactStoppedListener.ifPresent(ArtifactStoppedListener::onStart);
```

Comment:
no need to use the intermediate `Optional` just for this. a simple null check will do.